foundry.toml

```diff
@@ -2,7 +2,7 @@
 solc_version = "0.8.19"
 optimizer = true
 optimizer_runs = 200
-via_ir = false
+via_ir = true
```

Comment:
why do we need this? makes compilation a *lot* slower :(

Reply:
Yeah, makes compilation slower but won't compile without it:
```
Error:
Compiler run failed:
Error: Compiler error (/solidity/libsolidity/codegen/LValue.cpp:52):Stack too deep. Try compiling with `--via-ir` (cli) or the equivalent `viaIR: true` (standard JSON) while enabling the optimizer. Otherwise, try removing local variables.
   --> src/Manager.sol:513:21:
    |
513 |         sendMessage(recipientChain, encodedManagerPayload);
    |                     ^^^^^^^^^^^^^^

make: *** [Makefile:8: build] Error 1
```

Not sure if this is a machine-specific error or happens across the board.

Reply:
Can you refactor the relevant logic to block scope some variables instead?

Reply:
Can address this in a follow up PR -- https://github.com/wormhole-foundation/example-native-token-transfers/issues/29